.vscode/launch.json

```diff
@@ -67,20 +67,24 @@
             "name": "debug integration test",
             "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
             "env": {
-                "MSFT_TEST_DA_PORT": "4712",
+                // MSFT_TEST_DA_PORT: Enable this to debug the debug-adapter while the tests are running (The test will look for a debug-adapter server on port 4712 instead of launching a new one)
+                // "MSFT_TEST_DA_PORT": "4712",
+                // MSFT_TEST_TIMEOUT_MULTIPLIER: The tests have long timeouts for the CI server. While debugging locally it's sometimes useful to reduce the timeouts, to be able to run all the tests faster when they are failing. This value is a multiplier that will be applied to all timeouts using during the tests... (Work in progress)
+                "MSFT_TEST_TIMEOUT_MULTIPLIER": "0.4"
```

Comment:
I use it mostly to avoid possible collisions if someone wants to name their environment variable TEST_TIMEOUT_MULTIPLIER too... Do you see any disadvantage to have it?

Reply:
I don't think we need to worry about that

Reply:
Removed